onnx/large_helper.py

```diff
@@ -0,0 +1,333 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import enum
+import os
+from typing import Any, Dict, Iterable, Optional, Set, Tuple
+
+import numpy as np
+
+from onnx import (
+    GraphProto,
+    ModelProto,
+    StringStringEntryProto,
+    TensorProto,
+    checker,
+    load_model,
+)
```

Comment:
I would import modules only in any new code

Reply:
I missed we now need to follow that rule. Maybe a github action could check that.

Reply:
Right. This is harder to check last time I looked. I will add a CI check when I find a good way to do it. The motivation is https://gist.github.com/justinchuby/9085242a53158f2fd7ae7aa650e55ee3#3-import-only-modules